package.json

```diff
@@ -22,6 +22,8 @@
     "eslint": "eslint --max-warnings 0 --format codeframe --ignore-pattern '**/*.min.js' hq/public/javascripts/",
     "linter-check": "eslint --print-config .eslintrc.js | eslint-config-prettier-check",
     "prettier": "prettier -o --write './**/*.{js,css}'",
-    "sass-lint": "sass-lint -v -q hq/public/stylesheets/*.css"
+    "sass-lint": "sass-lint -v -q hq/public/stylesheets/*.css",
+    "snyk-test": "PATH=/path/to/sbt/executable/folder:$PATH snyk test --debug --org=guardian --json --file=build.sbt",
+    "snyk-monitor": "PATH=/path/to/sbt/executable/folder:$PATH snyk monitor --debug --org=guardian --file=build.sbt"
```

Comment:
Should we have Snyk listed in the devDependencies? I may have missed the convo - does Snyk ave to be installed globally, or can we use it as a local installation?

Reply:
If we do add Snyk to the devDeps

Can the update to the yarn.lock file also be included?

https://yarnpkg.com/lang/en/docs/yarn-lock/

It should auto-generate based on changes to package.json when you run `yarn install`

Reply:
Done.